Documentation/winpe-mmc.md

```diff
@@ -14,7 +14,7 @@ The high-level process we will follow is,
 
 First, we need to identify where the bootloader will live. i.MX6/7 can boot from a number of sources including eMMC/SD, NOR flash, SPI, I2C, and USB. For more information about i.MX6/7 early boot, see [Firmware Boot Documentation](boot.md) and the "System Boot" chapter of the processor reference manual. The reference BSP only implements boot from SD card. It is possible to port the reference material to boot from a different source, though for this document we will assume the initial boot device is SD. The initial boot device is set by processor fuses.
 
-To avoid bricking your device, we recommend putting the first stage bootloader on media that can be reprogrammed via external means if necessary, such as an SD card, SPI flash with external programing circuitry, or I2C flash with external programming circuitry. By external programming circuitry, we mean that you can write new contents to the storage device without booting the i.MX processor.
+To avoid breaking your device, we recommend putting the first stage bootloader on media that can be reprogrammed via external means if necessary, such as an SD card, SPI flash with external programing circuitry, or I2C flash with external programming circuitry. By external programming circuitry, we mean that you can write new contents to the storage device without booting the i.MX processor.
```

Comment:
bricking isn't a typo, it's standard terminology for making your computer as useful as a brick. 💻=🧱

Reply:
Thanks! I reverted the change.